content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Announcing Istio 1.9
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9!
+
+{{< tip >}}
+Officially supported versions of Kubernetes releases are 1.17 to 1.20 for this Istio release.
+{{< /tip >}}
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers `Shamsher Ansari` (Red Hat), `Steven Landlow` (Google) and `Jacob Delgado` (Aspen Mesh) for helping to get Istio 1.9.0
```

Comment:
Follow "We are pleased to announce" with the Day 2 paragraph, and move the supported versions and thank-yous to the end of the first section. They're not as important for a reader.

(Happy to do this in suggested changes after the outstanding suggestions are committed; right now it would probably stomp them if I did)

Reply:
Update. Please take another look.